transmogrifier/sources/xml/ead.py

```diff
@@ -557,6 +493,96 @@ def get_publishers(cls, source_record: Tag) -> list[timdex.Publisher] | None:
             return [timdex.Publisher(name=publication_value)]
         return None
 
+    @classmethod
+    def get_related_items(cls, source_record: Tag) -> list[timdex.RelatedItem] | None:
+        related_items = []
+        collection_description = cls._get_collection_description(source_record)
+
+        for related_item_element in collection_description.find_all(
+            ["altformavail", "relatedmaterial", "separatedmaterial"], recursive=False
+        ):
+            if related_item_element.name == "relatedmaterial":
+                related_items.extend(cls._get_related_material(related_item_element))
+            else:  # noqa: PLR5501
+                if related_item := cls.create_string_from_mixed_value(
```

Comment:
The lint ignore caught my attention, think it could be simplified to:

```python
if related_item_element.name == "relatedmaterial":
    related_items.extend(cls._get_related_material(related_item_element))
elif related_item := cls.create_string_from_mixed_value(
    related_item_element, separator=" ", skipped_elements=["head"]
):
```

Reply:
Yes ☝️

Reply:
Ah, I mistakenly thought that the statement for the `elif` condition would also execute if it was also met, so I hesitated to collapse the `if/else` conditions. This has been updated!